### PR TITLE
UICAL-226 - Jest for ServicePointAssignmentField

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose
create Jest/RTL test for components/fields/ServicePointAssignmentField.tsx

### CODE REVIEW

File: src/components/fields/ServicePointAssignmentField.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Form } from 'react-final-form';
+import '../../test/__mocks__/matchMedia.mock';
+import ServicePointAssignmentField from './ServicePointAssignmentField';
+import withIntlConfiguration from '../../test/util/withIntlConfiguration';
+
+const servicePoints = [
```
Comment (ncovercash) [inline src/components/fields/ServicePointAssignmentField.test.js]: When possible, please use the shared test service points.

You can import them with:

```js
import ServicePoints from '../../test/data/ServicePoints';
```

Then, use them like `<ServicePointAssignmentField servicePoints={ServicePoints} />`

(You can also do `import * as ServicePoints from ...`, then do `ServicePoints.SERVICE_POINT_1` and such, if you need finer control over which service points get used)

**Review by ncovercash — CHANGES_REQUESTED** [review]
Looks great overall, just use the predefined constant as mentioned in the comment.  Thanks for the help!

File: src/components/fields/ServicePointAssignmentField.test.js (hunk 1 above)
Comment (Terala-Priyanka) [reply src/components/fields/ServicePointAssignmentField.test.js]: Thank you, pushed the changes.

**Review by ncovercash — APPROVED** [review]
Looks great, thank you so much!

**Pull request merged**